package.json

```diff
@@ -91,14 +92,14 @@
         "enzyme": "^3.2.0",
         "enzyme-adapter-react-16": "^1.1.1",
         "gh-pages": "^2.0.0",
-        "husky": "^0.14.3",
-        "jest": "^22.0.3",
-        "lint-staged": "^7.0.4",
+        "husky": "^1.3.1",
+        "jest": "^24.0.0",
+        "lint-staged": "^8.1.1",
         "prettier": "^1.9.2",
-        "prettier-standard": "^8.0.1",
+        "prettier-standard": "^9.1.1",
         "prop-types": "^15.6.0",
-        "react": "16.2.0",
-        "react-dom": "16.2.0",
+        "react": "16.7.0",
+        "react-dom": "16.7.0",
```

Comment:
Thanks for fixing [this](https://github.com/rosedu/site-ng/network/alert/package.json/react-dom/open)

Reply:
Well actually. Everything breaks now due to new gatsby version. Will try to fix. Or just upgrade selectively.

Reply:
Haskell for the win :P

Reply:
Yeah... how about no.